index.html

```diff
@@ -0,0 +1,221 @@
+                <div class="context-projects-1">
+                    <button>See Project</button>
+                </div>
+                <div class="context-projects-2">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-2">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+            </div>
+
+        </section>
+        <section class="section-about">
+            <div class="about-text">
+                <h2>About me</h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
```

Comment:
Please add a little small margin between the projects and the About page. Kindly see the screenshot for reference.
![space](https://user-images.githubusercontent.com/35504149/135471315-4aefe96a-9686-4ffe-b001-cc235e08ce1c.png)